stanity/psisloo.py

```diff
@@ -0,0 +1,78 @@
+
+import psis
+import pandas
+import math
+import numpy
+import seaborn
+import matplotlib as pyplot
+
+class Psisloo(object):
+    def __init__(self, log_lik):
```

Comment:
A short description of what each of these attributes are in the docstring for the class would be helpful

Also I'd be biased toward non-abbreviated names (e.g. `log_likelihood`) unless you think the abbreviation is so standard that spelling it out would be more confusing (I haven't used the R libraries so I'm not sure if that is the case for these names)